### PR TITLE
feat: add QuickTemplate (Go) package

### PR DESCRIPTION
**Pull request by SharanSharathi**
This package adds source.qtpl syntax highlighting, code completion &
build system for sublime.

Contributing for this template-engine:
https://github.com/valyala/quicktemplate

### CODE REVIEW

**Comment by Ultra-Instinct-05** [comment]
This doesn't seem to be a ST package at all, some templating package for Go ? Why are you submitting it to PC ? I don't see any build system nor any syntax files nor anything in the README saying how to go about using the package in ST.

**Comment by Ultra-Instinct-05** [comment]
You have not made any tagged releases for your package. You need to at least release one [semantic versioned](https://packagecontrol.io/docs/submitting_a_package#Step_4) tagged release.

**Comment by SharanSharathi** [comment]
Thanks, I tried tagging 0.1.0 yesterday.
https://github.com/SharanSharathi/sublime_qtpl/releases/tag/0.1.0

Wasn't it visible? Should I add another tag?

**Comment by SharanSharathi** [comment]
Oh! Sorry, 
now I got it, I have to release the tag.

https://github.com/SharanSharathi/sublime_qtpl/releases/tag/0.1.0

**Comment by SharanSharathi** [comment]
Suggestion:
In the doc Submitting a Package [step 4](https://packagecontrol.io/docs/submitting_a_package#Step_4), it will be helpful to say, release the created tag.

now it says:
> you will need to "create a tag" each time you want to make a new version available to users

it can say:
> you will need to "create a tag and release it" each time you want to make a new version available to users

**Comment by SharanSharathi** [comment]
@Ultra-Instinct-05 , I've also fixed the warning from packagecontrol-bot,
and I've requested review from packagecontrol-bot, it didn't work, can you help?

File: repository/q.json [hunk 1]
```diff
@@ -359,6 +359,18 @@
 				}
 			]
 		},
+		{
+			"name": "QuickTemplate (Go)",
+			"details": "https://github.com/SharanSharathi/sublime_qtpl",
+			"issues": "https://github.com/SharanSharathi/sublime_qtpl/issues",
```
Comment (braver) [inline repository/q.json:365]: The issues url is the default url in GitHub so you don't need to (and therefore shouldn't) provide it here.

Comment (SharanSharathi) [reply repository/q.json]: Thanks for the feedback @braver , I've removed the issues url.

**Pull request merged**